Upgradeable Statistics stdlib (#46501)
JuliaLang/julia: NEWS.md

#### Dates

#### Statistics

> * Statistics has been promoted from being a standard library to a separate package. It now has to be explicitly installed to be used. However, functions `mean` and `mean!` are now exported directly by Base. ([#46501])

Reviewer: ```suggestion
* Statistics is now an upgradeable standard-library. The functions `mean` and `mean!` are now exported directly by Base. ([#46501])
```

#### Dates

#### Statistics

> * Statistics is now an upgradeable standard library.([#46501])

Reviewer 2: ```suggestion
* Statistics is now an upgradeable standard library ([#46501]).
```
